app/helpers/companies_helper.rb

```diff
@@ -4,5 +4,10 @@ def company_complete? company
     !company.nil? && !company.name.nil? && !company.name.empty? &&
       !company.region.nil? && !company.region.empty?
   end
-
+  def list_categories company
```

Comment:
I'm not sure what you're trying to accomplish.
This will just return the name of the last category for one single member application.

If a company has multiple categories, then what about all of the other categories?  IOW, what is so special about the _last_ category?

Why are you looking up a company by a membership application?
This is being used on the list of companies view.  Why not just get the information from each company directly?  ex: `company.categories` ?

I'm not sure exactly what you're trying to do, so I don't know you should be working with `company.categories` or not.  Can you clarify what it is that you want to see in PT?